src/video/wayland/SDL_waylandvideo.c

```diff
@@ -1114,6 +1115,8 @@ static void display_handle_global(void *data, struct wl_registry *registry, uint
     } else if (SDL_strcmp(interface, "kde_output_order_v1") == 0) {
         d->kde_output_order = wl_registry_bind(d->registry, id, &kde_output_order_v1_interface, 1);
         kde_output_order_v1_add_listener(d->kde_output_order, &kde_output_order_listener, d);
+    } else if (SDL_strcmp(interface, "zwp_pointer_gestures_v1") == 0) {
+        d->input->pointer_gestures = wl_registry_bind(d->registry, id, &zwp_pointer_gestures_v1_interface, 1);
```

Comment:
The input gestures interface is a global object, not seat specific, so it should be in the `SDL_VideoData` struct like the cursor shape manager and input timestamps manager.

Also, the global pointer gesture object needs to be destroyed when shutting down like the other Wayland objects.

Reply:
done